curation_concerns-models/app/actors/curation_concerns/file_set_actor.rb

```diff
@@ -107,11 +111,12 @@ def make_derivative(file_set_id, working_file)
         CharacterizeJob.perform_later(file_set_id, working_file)
       end
 
-      # @param [ActionDispatch::Http::UploadedFile] file
+      # @param [File, ActionDispatch::Http::UploadedFile] file
       # @param [String] id the identifer
       # @return [String] path of the working file
       def copy_file_to_working_directory(file, id)
-        copy_stream_to_working_directory(id, file.original_filename, file)
+        name = file.respond_to?(:original_filename) ? file.original_filename : ::File.basename(file)
```

Comment:
why don't we use `file_set.label` here (or pass it as a parameter)?  Then we avoid repeating this test.